exporter/exporterhelper/internal/producer_consumer_queue.go

```diff
@@ -1,32 +1,45 @@
 // Copyright The OpenTelemetry Authors
+// Copyright (c) 2019 The Jaeger Authors.
```

Comment:
Why this change?

Reply:
This is derivative of Jaeger's `BoundedQueue` (though after refactoring and breaking down into interface and implementation). Should it be removed then?

Reply:
The interface was not copied from Jaeger so no need to add the new copyright I believe. Not sure how this works.

Reply:
Yeah, I wasn't fully confident about it either. @mx-psi what do you think?

Reply:
I am not really related to the Jaeger project (just happened to contribute to `pkg/queue` a while back) nor do I know enough about copyright law to answer this, sorry 😄